### ISSUE TITLE
Instant commands on Paper 1.20.2

### ISSUE THREAD

**Issue by MrMaleficus** [Azuriom/AzLink#29]
Hello,

I have this reccuring error in the minecraft server console : 

```
[17:06:45 WARN]: [io.netty.channel.ChannelInitializer] Failed to initialize a channel. Closing: [id: 0x81ca4ad8, L:/172.45.0.4:25588 - R:/172.45.0.1:50970]
java.lang.NoClassDefFoundError: io/netty/handler/codec/http/HttpRequestDecoder
        at com.azuriom.azlink.common.http.server.HttpChannelInitializer.initChannel(HttpChannelInitializer.java:22) ~[AzLink.jar:?]
        at com.azuriom.azlink.common.http.server.HttpChannelInitializer.initChannel(HttpChannelInitializer.java:11) ~[AzLink.jar:?]
        at io.netty.channel.ChannelInitializer.initChannel(ChannelInitializer.java:129) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.ChannelInitializer.handlerAdded(ChannelInitializer.java:112) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:1114) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:609) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.DefaultChannelPipeline.access$100(DefaultChannelPipeline.java:46) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.DefaultChannelPipeline$PendingHandlerAddedTask.execute(DefaultChannelPipeline.java:1463) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.DefaultChannelPipeline.callHandlerAddedForAllHandlers(DefaultChannelPipeline.java:1115) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.DefaultChannelPipeline.invokeHandlerAddedIfNeeded(DefaultChannelPipeline.java:650) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:514) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569) ~[netty-transport-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) ~[netty-common-4.1.97.Final.jar:4.1.97.Final]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassNotFoundException: io.netty.handler.codec.http.HttpRequestDecoder
        ... 21 more
```

Commands seems to be currently passed, but this error occurs since the 1.20.2 minecraft update.
I'm using paper, latest version.

**Comment by MrMicky-FR** [comment]
For reference, this is caused by https://github.com/PaperMC/Paper/commit/e57af7d01e509c747520cc1037169029256289bf#diff-afd402e1f2786f960e8a884624d91bbc9d55475f90952f15f36c55fc6d39d583L28-L37 and only affects instant commands.

Until we find a solution, using AzLink Legacy should solve this issue

**Comment by MrMicky-FR** [comment]
Fixed in [AzLink v1.3.2](https://github.com/Azuriom/AzLink/releases/tag/v1.3.2)
